test/fixtures/dialects/snowflake/snowflake_alter_table_column.yml

```diff
@@ -202,18 +377,3 @@ file:
       - column_reference:
           identifier: my_column
 - statement_terminator: ;
-- statement:
```

Comment:
FYI - I moved this to the top of the file, so the order of test cases in this file aligns with the order in code.